### PR TITLE
setRouter on all AngleRouter

### PR DESCRIPTION
**Pull request by GuillaumeNervoXS**

### CODE REVIEW

File: scripts/foundry/router/RouterSetAggregator.s.sol [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { IERC721Metadata } from "oz/token/ERC721/extensions/IERC721Metadata.sol";
+import { Enum } from "safe/Safe.sol";
+import { MultiSend, Utils } from "../Utils.s.sol";
+import "../Constants.s.sol";
+
+interface IAngleRouter {
+    function setRouter(address router, uint8 who) external;
+
+    function oneInch() external returns (address);
+}
```
Comment (0xtekgrinder) [inline scripts/foundry/router/RouterSetAggregator.s.sol:14]: maybe move this inside interface folder

File: scripts/foundry/router/RouterSetAggregator.s.sol [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            bytes memory data = abi.encodeWithSelector(IAngleRouter.setRouter.selector, aggregator, 1);
+            additionalData = abi.encode(aggregator);
+            uint256 dataLength = data.length;
+            address to = angleRouter;
+            bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+            transactions = abi.encodePacked(transactions, internalTx);
+        }
+
+        bytes memory payloadMultiSend = abi.encodeWithSelector(MultiSend.multiSend.selector, transactions);
```
Comment (0xtekgrinder) [inline scripts/foundry/router/RouterSetAggregator.s.sol:39]: we can just add the data as raw without multisend

**Review by 0xtekgrinder — COMMENTED** [review]
just a few non mandatory changes

File: scripts/foundry/router/RouterSetAggregator.s.sol (hunk 1 above)
Comment (GuillaumeNervoXS) [reply scripts/foundry/router/RouterSetAggregator.s.sol:14]: There is no contract folder

**Review by sogipec — APPROVED** [review]

**Pull request merged**